ztp/source-crs/MachineConfigContainerMountNS.yaml

```diff
@@ -1,30 +1,28 @@
+# Automatically generated by source-crs-builder
+# Do not make changes directly.
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
-  name: container-mount-namespace-and-kubelet-conf
   labels:
     machineconfiguration.openshift.io/role: master
+  name: container-mount-namespace-and-kubelet-conf-master
```

Comment:
Why limited to just master? Wouldn't this be valuable on all nodes?

Reply:
It would be, but that's a problem to be solved in another PR for all MC objects.